src/main/java/org/jboss/remoting3/Endpoint.java

```diff
@@ -379,6 +380,8 @@ static Endpoint getCurrent() {
      */
     IoFuture<Connection> connect(URI destination, InetSocketAddress bindAddress, OptionMap connectOptions, SSLContext sslContext, AuthenticationConfiguration connectionConfiguration);
 
+    IoFuture<Connection> connect(final URI destination, final OptionMap connectOptions, final CallbackHandler callbackHandler) throws IOException;
```

Comment:
These don't need to be final.  IDEs add it sometimes; it's not clear why though :)

Reply:
Updated

Reply:
Does not seem to be?